src/bundle.rs

```diff
@@ -550,10 +601,9 @@ pub fn export_with_super_mechanisms(lems: &LemsFile, nml: &[String], bundle: &st
                 // and we sum currents as sum(g_)
                 let i = mechs
                     .iter()
-                    .map(|m| format!("{m}_conductance*(v - {m}_e{ion})"))
+                    .map(|m| format!("{m}_g*(v - {m}_e{ion})"))
```

Comment:
The Ih channel now does the same thing between normal and supermechs in my branch :rocket:

Reply:
:tada: